exercises/lesson_05/breach/README.rst

```diff
@@ -0,0 +1,26 @@
+==========
+Assignment
+==========
+
+Your task is to examine and act upon a reported security breach on one of your
+systems. Should you need it, the ASSIGNMENT file is located under the data
+directory.
+
+How to run this lab
+-------------------
+
+::
+
+    # Setup the environment for this lab    (requires root)
+    $ sudo ./lab setup
+
+    # Watch the wonderful intro
+    $ ./lab intro
+
+    # Resolve the tasks
+
+    # Grade your work
+    $ sudo ./lab grade
+
+    # Clean the environment that was created for the lab    (requires root)
+    $ sudo ./lab cleanup
```

Comment:
Would be nice to add info, how to end the lab (if possible) if you either cannot or don't want to finish the lab.